src/gt4py/backend/dawn_backends.py

```diff
@@ -467,54 +473,45 @@ class BaseDawnBackend(gt_backend.BasePyExtBackend):
 
     _field_info = {}
 
-    @classmethod
-    def generate(
-        cls,
-        stencil_id: gt_definitions.StencilID,
-        definition_ir: gt_ir.StencilDefinition,
-        definition_func: types.FunctionType,
-        options: gt_definitions.BuildOptions,
-        **kwargs,
-    ):
+    def generate(self, **kwargs):
         from gt4py import gt_src_manager
 
-        cls._check_options(options)
+        self._check_options(self.builder.options)
+
+        implementation_ir = gt_analysis.transform(self.builder.definition_ir, self.builder.options)
```

Comment:
This is not something I only introduced because tests failed without it (even on master at that point).

Reply:
This call to the analysis transform does not exist on master (https://github.com/GridTools/gt4py/blob/e9fe299319c8d12a28bbc45135da76c427d7fa23/src/gt4py/backend/dawn_backends.py#L473)

If it appears here, it indicates that there are still merge conflicts and at least this file is based on the wrong commit.

Reply:
The call does not exist indeed, however the tests for dawn backends fail on master due to it's absence. Since these tests are not part of the CI now, I will remove this line and the dawn backends must be fixed separately.